package.json

```diff
@@ -67,14 +67,15 @@
     "typescript": "^4.7.4"
   },
   "scripts": {
-    "build": "yarn clean && yarn build:generate && yarn build:esm && yarn build:subpaths && yarn build:cjs",
+    "build": "yarn clean && yarn build:generate && yarn build:esm && yarn build:subpaths && yarn build:cjs && yarn build:fed:packages",
```

Comment:
Are we sure we don't want this to be opt-in only? Are there circumstances where this would be an additional, unnecessary build step for users?

Reply:
Unfortunatly we need to release this and i'm not sure of a way to build it after a release except including the script.  I don't think there will be any harm in adding the additional package.json files.  tree shaking should take care of it when they build there apps.